target/linux/qualcommax/ipq807x/base-files/etc/hotplug.d/firmware/11-ath11k-caldata

```diff
@@ -52,6 +50,21 @@ case "$FIRMWARE" in
 		ath11k_patch_mac $(mtd_get_mac_binary boarddata1 0x6) 2
 		ath11k_set_macflag
 		;;
+	netgear,wax620)
+		caldata_extract "0:art" 0x1000 0x20000
+		label_mac=$(get_mac_label)
+		ath11k_patch_mac $(macaddr_add $label_mac -31) 1
+		ath11k_patch_mac $(macaddr_add $label_mac 1) 0
```

Comment:
Is the order changed by mistake? MAC with id 0 is 2.4G address.

Reply:
So you have:
```
WLAN5 = LAN1 - 0x21
WLAN2.4 = LAN1 - 0x01
```
which is not correct.

Reply:
Do we agree on the changes @testuser7 ? So we can give robi a go for merge.

Reply:
Yes, you're right, I mixed up the order earlier.